modules/light-clients/07-tendermint/store_test.go

```diff
@@ -92,12 +92,10 @@ func (suite *TendermintTestSuite) TestGetConsensusState() {
 }
 
 func (suite *TendermintTestSuite) TestGetProcessedTime() {
-	// setup
 	path := ibctesting.NewPath(suite.chainA, suite.chainB)
-
 	suite.coordinator.UpdateTime()
-	// coordinator increments time before creating client
-	expectedTime := suite.chainA.CurrentHeader.Time.Add(ibctesting.TimeIncrement)
```

Comment:
change in testing pkg behaviour, this no longer occurs

Reply:
https://github.com/cosmos/ibc-go/pull/4535#discussion_r1314802915 lol